routes/session.js

```diff
@@ -31,5 +31,25 @@ exports.create = function(req, res) {
 		if (err) return console.err(err);
 
 		console.log(sessionObject);
+		Session.findOne( {_id: req.classId}), function (err, session) {
```

Comment:
I'm not totally sure on this one, but I think that this may need to be `Class.findOne` rather than `Session.findOne` since we'll be searching through for the classes to find the one that we're looking for. What do you think @kevinliao95 ?